### ISSUE TITLE
Remote instance able to delete local comments?

### ISSUE THREAD

**Issue by Quix0r** [friendica/friendica#13357]
- [x] I have searched open and closed issues for duplicates

### Bug Description

A remote user has posted a comment I commented on. When he deleted my comment there, it got also deleted from my timeline as. This shouldn't happen as I own the comment (but not the post, of course). Please don't mind the discussion there: https://loma.ml/display/373ebf56-1764-de3b-afbb-e3c325830121

While it should be still available on my instance:
https://f.haeder.net/screenshot.png

### Steps to Reproduce

1.  Create a post on a remote Friendica instance you follow
2.  Comment on an other instance on that post
3.  Delete the comment on the remote site.

Actual Result:

My comment got deleted on my instance as well.

Expected Result:

My comment remains intact on my instance while it is deleted on remote site.

### Screenshots

https://f.haeder.net/screenshot.png

PS: Sorry, upload with Github always fails.

### Platform Info

Friendica Version: `2023.09-dev`

Friendica Source: `git`

PHP version: `7.4.11` (not relevant)

SQL version: `10.5.19-MariaDB`

**Comment by MrPetovan** [comment]
What makes you think you own your comments on other people's conversations?

**Comment by Quix0r** [comment]
I have made the comment locally on my instance. So I should own that comment because I made it. Sure when they delete it there, I cannot do anything about it. But: It got deleted on  my end as well and that's what is confusing me.

**Comment by MrPetovan** [comment]
Comment ownership is completely different between ActivityPub and Diaspora. In ActivityPub, each comment is owned by its author as it has an existence outside of the conversation it's part of. In Diaspora the ownership of the comment is relinquished to the top-level author as it doesn't have an existence outside of the conversation.

Since Friendica supports both protocols, I can see it going either way, possibly differently depending on the context. Since Friendica is more geared towards conversations, I'm not surprised the behavior between Friendica nodes would be based on the Diaspora philosophy.

**Comment by Quix0r** [comment]
Yes, I'm the author of my comments and the other account was able to deleted comments on my instance I have authored.
